# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mtcps/no_manpage.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN mtcps-0.2.orig/Makefile mtcps-0.2/Makefile
--- mtcps-0.2.orig/Makefile	2002-12-03 16:37:27.000000000 +0100
+++ mtcps-0.2/Makefile	2005-07-23 01:18:13.000000000 +0200
@@ -19,7 +19,6 @@
 
 install: mtcps
 	cp mtcps $(BINPATH)
-	cp mtcps.8 $(MANPATH)/man8
 
 clean:
 	rm -f *~ *.o mtcps
